src/routes/rooms.ts

```diff
@@ -49,10 +49,18 @@ router.post(
   asyncHandler(async (req, res) => {
     const body: CreateRoom = req.body;
 
-    if (!body || Array.from(body.options).length < 1) {
-      res.status(HttpStatus.UNPROCESSABLE_ENTITY);
-      res.send();
-      return;
+    if (!body) {
+      throw new UnprocessableEntityError(
+        'ROOM_CREATE_EMPTY_DATA',
```

Comment:
nit: we should move constant names into their own file for easier refactor.

Reply:
I totally agree, I didn't move them to a constants file because each error code is unique, so they most likely won't be reusable, maybe we can add them as constants within the file, what do you think?

Reply:
I would still move them to a different file. 
Changes in the future will be easier to action if we do that (still a nit though)

Reply:
ok, we can try that, do you want me to create a global constants file, something scoped to the route or maybe something along the lines of `rooms.errors.ts`?

Just wondering, how by moving them to a constants file can make it easier to act on?

Reply:
Answer 1: a scoped file seems better.
Answer 2: it’s easier to search/update a constant in a file full of constants than update a string that lives inside a file full of imports/functions/global variables/exports. That’s a personal opinion though. Also variables are easier to navigate than Strings (navigate to declaration, etc)

Reply:
> Answer 2: it’s easier to search/update a constant in a file full of constants than update a string that lives inside a file full of imports/functions/global variables/exports. That’s a personal opinion though. Also variables are easier to navigate than Strings (go to declaration, find usages, etc)

I totally agree, it's just that this case is a little bit special, since these constants are scoped to a route and their value it won't probably be reused, that's why I think it's fine declaring them in the same file

Reply:
(This is only a ```nit``` btw 😆 ) . 

But my point wasn't reusability. Imagine this file grows a lot in the future with many different routes, variables, etc. I feel like updating a String there *could* be harder than update a String in a file with 4 or 5 lines of code.

Reply:
totally agree, I just don't want to over-engineer things now since this error handling strategy it might not be definitive, declaring them in the top of the file it's fine for now